Makefile

```diff
@@ -231,6 +231,7 @@ install-deps: $(ALERTMANAGER) $(MINIO) $(PROMETHEUS_ARRAY)
 	@echo ">>GOBIN=$(GOBIN)"
 
 .PHONY: docker-ci
+docker-ci: export BIN_DIR= $(shell pwd)/tmp/bin
```

Comment:
Why do we need this if you are replacing `BIN_DIR` with `GOBIN`?

Reply:
No idea, red herring - although relying on yet another bin dir seems wrong